cmd/forwarder/fixer.go

```diff
@@ -56,6 +63,13 @@ func fix(r io.Reader, remoteAddr string, logplexDrainToken string) ([]byte, erro
 			messageWriter.Write(b)
 		}
 
+		if authUser != "" {
+			messageWriter.WriteString(" log_iss_user=" + authUser)
+			if replace {
+				messageWriter.WriteString("\n")
```

Comment:
@freeformz I considered just always removing `\n` (if present via `TrimRight`, it's a couple of ns faster to skip the `HasSuffix` check), but I wasn't 100% sure it's safe to always add a `\n` to the end, so opted for the safe method. Any guidance here?

Reply:
Always trimming `\n` should be fine since the messages are length prefixed.